### PR TITLE
Preparatory courses no longer have 'credits' in title, but 'fup' instead

### PR DESCRIPTION
**Pull request by axelbjo**
Passing the course_level_code property from courseData to determine whether or not a course should have 'credits'/'hp' or 'fup' in the title.

### CODE REVIEW

**Review by amirhossein-haerian — COMMENTED** [review]
LGTM ⭐️

**Review by amirhossein-haerian — APPROVED** [review]

File: public/js/app/components/CourseTitle.jsx [hunk 1]
```diff
@@ -6,8 +6,10 @@ import { useFormatCredits } from '../hooks/useFormatCredits'
 
 const adminLink = (courseCode, languageShortname) => `/kursinfoadmin/kurser/kurs/${courseCode}?l=${languageShortname}`
 
-const CourseTitle = ({ courseTitleData = '', pageTitle }) => {
+const CourseTitle = ({ courseTitleData = '', pageTitle, preparatory }) => {
```
Comment (karlandindrakryggen) [inline public/js/app/components/CourseTitle.jsx:9]: Name the new prop courseLevelCode instead? Becuase `preparatory` is just one possible value?
```suggestion
const CourseTitle = ({ courseTitleData = '', pageTitle, courseLevelCode }) => {
```

File: public/js/app/components/CourseTitle.jsx [hunk 2]
```diff
@@ -6,8 +6,10 @@ import { useFormatCredits } from '../hooks/useFormatCredits'
...
   const title = courseTitleData
+  // const preparatory = courseData.courseInfo.course_level_code
```
Comment (karlandindrakryggen) [inline public/js/app/components/CourseTitle.jsx:11]: Remove comment
```suggestion
```

**Review by karlandindrakryggen — APPROVED** [review]
Just rename the prop and remove commented-out code

**Pull request merged**